### DISCUSSION TITLE
Mysql scheme not sequential id.

### DISCUSSION THREAD

**Discussion by zLupa** [adonisjs/core#1595] [Help]
How do i set a random id or UUID in a scheme?

**Comment by prashanth1k** [comment]
For primary id or some random column?

UUID for primary key - see [primarykey](https://adonisjs.com/docs/4.1/lucid#_primarykey) and [incrementing](https://adonisjs.com/docs/4.1/lucid#_incrementing). No clue whether it works in v5, sorry :)

For any random column: use a trigger or expression. You could consider setting `uuid` from the application.
